sirepo/pkcli/job_agent.py

```diff
@@ -536,6 +536,8 @@ def _create_in_file(self):
 
 class _FastCgiCmd(_Cmd):
     def destroy(self):
+        pkio.unchecked_remove(self.dispatcher.fastcgi_file)
```

Comment:
This function should be in the dispatcher, e.g. self.fastcgi_destroy.

Also the file may be none, which I don't think is supported by unchecked_remove so check to see if it is not none.

Reply:
https://github.com/radiasoft/sirepo/pull/2809/commits/e0b28e875b68a055a197fc4c738bd3f3107fe01e